akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -484,17 +495,32 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
         // FIXME we should call FragmentAssembler.freeSessionBuffer when image is unavailable
       }
     })
+
     ctx.errorHandler(new ErrorHandler {
+      private val fatalErrorOccured = new AtomicBoolean
+
       override def onError(cause: Throwable): Unit = {
         cause match {
-          case e: ConductorServiceTimeoutException ⇒
-            // Timeout between service calls
-            log.error(cause, s"Aeron ServiceTimeoutException, ${cause.getMessage}")
-
+          case e: ConductorServiceTimeoutException ⇒ handleFatalError(e)
+          case e: DriverTimeoutException           ⇒ handleFatalError(e)
+          case _: AeronTerminated                  ⇒ // already handled, via handleFatalError
           case _ ⇒
             log.error(cause, s"Aeron error, ${cause.getMessage}")
         }
       }
+
+      private def handleFatalError(cause: Throwable): Unit = {
+        if (fatalErrorOccured.compareAndSet(false, true)) {
+          if (!isShutdown) {
+            log.error(cause, "Aeron {}, terminating system. {}", Logging.simpleName(cause), cause.getMessage)
+            taskRunner.stop()
+            aeronErrorLogTask.cancel()
+            system.terminate()
+            throw new AeronTerminated(cause)
+          }
+        } else
+          throw new AeronTerminated(cause)
+      }
```

Comment:
So it would be possible to recover this by starting a new actor system in the same JVM, but the transport lifecycle is bound to the actor system so it would not be possible to restart the transport alone?